[Bulk] Broken Link fixes (#878)
MicrosoftDocs/biztalk-docs: biztalk/adapters-and-accelerators/accelerator-swift/step-1-installing-the-base-platform.md

- Make sure that you enter the appropriate licensing information (maximum number of connections you have purchased per server). System performance can be affected by the number of available connections.

> - Make sure that you have installed all the software prerequisites required for a BizTalk Server installation. For more information, see the BizTalk Server Installation Instructions at [Installation Guide for BizTalk 2013 R2 Accelerator for SWIFT](/biztalk/adapters-and-accelerators/accelerator-swift/install-biztalk-accelerator-for-swift).

Reviewer: ```suggestion
- Make sure that you have installed all the software prerequisites required for a BizTalk Server installation. For more information, see the [BizTalk Server What's New, Install, Configuration, and Upgrade](/biztalk/install-and-config-guides/biztalk-server-what-s-new-installation-configuration-and-upgrade) and [Install BizTalk Accelerator for SWIFT](/biztalk/adapters-and-accelerators/accelerator-swift/install-biztalk-accelerator-for-swift).
```

- Test all critical updates in an offline environment before installing them on your production servers.

- Make sure that you install all the applicable hotfixes for all the products that you install as part of your deployment. For more information, see the following sources:

  - Microsoft BizTalk Server Online Help

>   - [BizTalk Server Installation Instructions](https://go.microsoft.com/fwlink/?LinkId=81041).

Reviewer: ```suggestion
  - [BizTalk Server Installation Instructions](https://go.microsoft.com/fwlink/?LinkId=81041)
```

>   - [BizTalk Server Installation Instructions](https://go.microsoft.com/fwlink/?LinkId=81041)

Reviewer: FW link doesn't go to intended destination:

```suggestion
  - [BizTalk Serve What's New, Install, Configuration, and Upgrade](/biztalk/install-and-config-guides/biztalk-server-what-s-new-installation-configuration-and-upgrade)
```

>   - Microsoft [!INCLUDE[btsWinNoVersion](../../includes/btswinnoversion-md.md)] Web site

Reviewer: ```suggestion
  - Microsoft website [!INCLUDE[btsWinNoVersion](../../includes/btswinnoversion-md.md)]
```

>   - [Microsoft Download Center Web site](https://www.microsoft.com/download)

Reviewer: ```suggestion
  - [Microsoft Download Center](https://www.microsoft.com/download)
```
